app.py

```diff
@@ -1,31 +1,29 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
```

Comment:
This file is no longer used (along with `flask_config.py` and the SECRET_KEY variable)

This also breaks the setup as `python-dotenv` has been removed from the dependency list